docs/UserGuide.md

```diff
@@ -248,18 +252,18 @@ Edits an existing student in the list.
     </div>
 
 3. Amount Paid (`paid/`):
-    - Amount Paid stands for the amount that has been paid by the Student.
-    - Amount Paid can only be an integer.
-    - Amount Paid must be non-negative.
+    - Amount Paid stands for the amount that has been paid by the student.
+    - Amount paid can only be an integer.
+    - Amount paid must be non-negative.
 
 4. Amount Owed (`owed/`):
-    - Amount Owed stands for the amount that has been owed by the Student.
+    - Amount Owed stands for the amount that has been owed by the student.
     - Format: `owed/` must be followed by an integer.
     - Amount Owed can only be an integer.
     - Amount Owed must be non-negative.
 
 5. Rates per Class (`rate/`):
-    - Rates per Class stands for the amount of charge per Class for the Student.
```

Comment:
Is this better or is 
`Rates per Class stands for the amount of charge per class for the student.` better?